docs/api.rst

```diff
@@ -1495,6 +1495,9 @@ of :class:`enum.Enum`.
     .. attribute:: discord_certified_moderator
 
         The user is a Discord Certified Moderator.
+    .. attribute:: http_interactions_bot
+
+        The user is a bot which receives interactions over HTTP.
```

Comment:
Should probably make the docs be more similar to what it says at https://github.com/discord/discord-api-docs/blob/8f435d11d78ea1c3db167364befc733df860bdcc/docs/resources/User.md?plain=1#L80

```suggestion
    .. attribute:: bot_http_interactions

        The user is a bot that uses only HTTP interactions and is shown in the online member list.
```